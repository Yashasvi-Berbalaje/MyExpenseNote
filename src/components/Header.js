import {NavLink} from 'react-router-dom';
import React from 'react';
 
  const Header = () =>(
    <div>
    <h1>Expensify</h1>
    <NavLink to='/Dashboard' activeClassName='is-active'>Home</NavLink>
    <NavLink to='/expense' activeClassName='is-active'>AddExpense</NavLink>
    </div>
  );
  export default Header;