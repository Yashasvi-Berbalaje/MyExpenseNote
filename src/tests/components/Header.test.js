
import Header from '../../components/Header';
import {shallow} from 'enzyme';
import toJSON from 'enzyme-to-json';
import React from 'react';

test('Should test for Header',()=>{
    const wrapper = shallow(<Header/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
})