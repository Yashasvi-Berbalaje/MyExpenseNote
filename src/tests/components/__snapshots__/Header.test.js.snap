// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test for Header 1`] = `
<div>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    to="/Dashboard"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/expense"
  >
    AddExpense
  </NavLink>
</div>
`;
